scipy/special/_special_ufuncs_docs.cpp

```diff
@@ -4762,6 +4762,42 @@ const char *mathieu_cem_doc = R"(
     --------
     mathieu_a, mathieu_b, mathieu_sem
 
+    Notes
+    -----
+    Even Mathieu functions are the solutions to Mathieu's differential equation
```

Comment:
```suggestion
    The even Mathieu functions are the solutions to Mathieu's differential equation
```

Otherwise, it's easy to start reading with emphasis on *Even*, which quickly gets confusing.
I'm not a domain expert, so please verify.